changelog/5040.bugfix.rst

```diff
@@ -0,0 +1,5 @@
+Change dependency Networkx from featurizers ``setup.py`` and ``requirements.txt``.
+
+There is an imcompatibility between Rasa dependecy requests 2.22.0 and the own depedency from Rasa for networkx raising errors upon pip install. There is also a bug corrected in ``requirements.txt`` which used ``~=`` instead of ``==``. All of these are fixed using networkx 2.4.0.
```

Comment:
do you know what the conflicting dependency is?

Reply:
Yes @tmbo, Rasa and requests 2.22.0 (as written in the changelog).

Reply:
mhm but requests does not have a dependency on networkx

Reply:
@tmbo you are right, I checked the dependencies for requests, but it also depends on some other packages that may depend on networkx. Somehow pip can't resolve this dependencies and gives that error. But with this PR, it will vanish =)